src/Compilers/VisualBasic/Test/Syntax/IncrementalParser/IncrementalParser.vb

```diff
@@ -772,6 +772,32 @@ End Module
         Assert.Equal(False, incrementalTree.GetRoot().ContainsDiagnostics)
         VerifyEquivalent(incrementalTree, expectedTree)
     End Sub
+
+    <Fact>
+    <WorkItem(405887, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=405887")>
+    Public Sub IncrementalParseInterpolationInSingleLineIf()
+        ' The code below intentionally is missing a space between the "Then" and "Console"
```

Comment:
I didn't understand this comment. There is a space between the "Then" and "System.Console...." below.